docs/user/openstack/connect_nodes_to_manila_network.md

```diff
@@ -0,0 +1,35 @@
+# Connecting nodes to a dedicated Manila network
+
+If you are going to use OpenStack Manila on your cluster, you should be aware that the service usually uses a standalone network for its shares, and by default the installer does not connect nodes to this network. It causes problems with mounting shares on an already deployed cluster, as there is no route to the network:
```

Comment:
Not sure we need to detail the different cases where Manila uses an additional network. Perhaps we could phase it like:

```suggestion
Depending on your OpenStack platform configuration, it is very likely that Manila uses a dedicated network for its shares. When that is the case, you need to attach your OpenShift compute nodes to this network otherwise pods might fail to mount the shares, as shown in the following example:
```

Also add something like `Consult with your OpenStack administrator to know what network Manila exposes its shares on.` after the example error message.

Reply:
Please see also the second part of my comment.